code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -2243,11 +2243,14 @@ obj/item/device/pda/AltClick()
 		T.date = current_date_string
 		T.time = worldtime2text()
 		id.virtual_wallet.transaction_log.Add(T)
-
-		to_chat(user, "<span class='info'>You insert [dosh] into the PDA.</span>")
-		qdel(dosh)
-		updateUsrDialog()
-
+		if(T.amount == 1)
+			to_chat(user, "<span class='info'>You insert [T.amount] credit into the PDA.</span>")
```

Comment:
you could use something like `You insert [T.amount] credit[T.amount == 1 ? "" : "s"]` instead. Actually, isn't there a `\s` macro already?

Reply:
Yes, BYOND's `\s` macro should do wonders here.

Reply:
Oh cool, I asked in the Discord but no one knew how. I remember we had some way to do it but not how.